tests/integration/createBirthdaySquad.test.ts

```diff
@@ -0,0 +1,77 @@
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import { afterEach, beforeAll, describe, expect, it, vi } from "vitest";
+
+import { squadJoins, users } from "@/db/schema";
+import { timeout } from "@/testUtils/constants";
+import { deleteLastDm } from "@/testUtils/integration/deleteLastDm";
+import { sendScheduleEvent } from "@/testUtils/integration/sendScheduleEvent";
+import { waitForDm } from "@/testUtils/integration/waitForDm";
+import { testDb } from "@/testUtils/testDb";
+import { scheduleEvent } from "@/types/schedule";
+
+dayjs.extend(utc);
+
+const testSlackTeamId = import.meta.env.VITE_SLACK_TEAM_ID;
+const testUserId = import.meta.env.VITE_SLACK_USER_ID;
+const testBotUserId = import.meta.env.VITE_SLACK_BOT_USER_ID;
+
+const constants = vi.hoisted(() => ({
+  birthdayPerson: "U001",
+  eventId: "E001",
+  teamId: testSlackTeamId,
+  otherUserIds: [testUserId, testBotUserId],
+  converastionId: "CH001",
+}));
+
+describe("CreateBirthdaySquad", () => {
+  beforeAll(async () => {
+    await testDb.delete(users);
+    await testDb.delete(squadJoins);
```

Comment:
I don't think we are touching squadJoins here. It might be cleaner to create a util that wipes all tables because it's a bit of a mess right now. What do you think?

Reply:
I think thats fine if we just take care what we want to clean up. I removed the squadJoins cleanup from here